cache/lru_cache.cc

```diff
@@ -214,11 +220,23 @@ double LRUCacheShard::GetHighPriPoolRatio() {
   return high_pri_pool_ratio_;
 }
 
+double LRUCacheShard::GetLowPriPoolRatio() {
+  DMutexLock l(mutex_);
+  return low_pri_pool_ratio_;
+}
+
 void LRUCacheShard::LRU_Remove(LRUHandle* e) {
   assert(e->next != nullptr);
   assert(e->prev != nullptr);
   if (lru_low_pri_ == e) {
-    lru_low_pri_ = e->prev;
+    if (e->prev != lru_bottom_pri_) {
+      lru_low_pri_ = e->prev;
+    } else {
+      lru_low_pri_ = &lru_;
+    }
```

Comment:
Could you clarify why we have to branch here?

Reply:
If lru_low_pri_'s pool still has some entries, we can simply assign `e->prev` to `lru_low_pri_`. However, if the current `e` is the last entry in the lru_low_pri_'s pool, then `e->prev` == lru_bottom_pri_, which means  `lru_low_pri_` has to point back to the original head `lru`, since  lru_low_pri_'s pool is empty now.

Reply:
Yes, I will add some comments in the codebase

Reply:
Not sure I'm following. `lru_low_pri_` denotes the head of the low-pri pool, right? If the cache currently only contains high and bottom priority entries, shouldn't we make it actually point to `lru_bottom_pri_` so any new low-prio entries get inserted in the correct position?

Reply:
You are right, we can use this way to simplify the code. The reason my way also works is because I handled this case in `Insert()`. I'll update it.

Reply:
DONE